### ISSUE TITLE
BUGS!!

### ISSUE THREAD

**Issue by aryankhatana01** [aryankhatana01/hindi-gpt-mini#1]
This has some unidentified bugs because of which the generation is a subset of the original dataset - need to figure out the bug.
Some possibilities are :
- Quantity of the training data
- Val loss not decreasing after a point
- hyperparameters

**Comment by aryankhatana01** [comment]
The problem is the dataset, it's repeating and very small, overfitting the model. Need to get another dataset :(

**Comment by aryankhatana01** [comment]
Might scrape data from this [website](http://premchand.co.in/stories) with Munshi Premchand's stories.
